frontend-react/src/pages/submissions/SubmissionTable.tsx

```diff
@@ -37,6 +39,7 @@ const SubmissionTableContent: React.FC<SubmissionTableContentProps> = ({
     const transformDate = (s: string) => {
```

Comment:
Oh, also!  Unrelated and I realize it existed beforehand so we can make this change in a separate pull request, but this function could be moved out since it's getting recreated on every render.